### PR TITLE
Set Up a "Hello world" Project With Two Apps

### PR DESCRIPTION
**Pull request by DJ-MrJay**
Here, I:
Created a new React app called 'hello-react-front-end'
Initialized project with Git.
Created the `App` component with react-router.
Created the `Greeting` component that will display a greeting. Set it up as a route in the `App` component.
Created a store, an action and a reducer that connect to the API endpont at `http://localhost:3000/` to get a random greeting.
Displayed the random greeting in the `Greeting` component.

### CODE REVIEW

File: src/Greeting.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchGreeting } from './redux/actions';
+
+const Greeting = () => {
+  const dispatch = useDispatch();
+  const { randomGreeting } = useSelector((state) => state.greeting.randomGreeting);
```
Comment (Davidosky007) [inline src/Greeting.js]: * Kindly adjust your code here the variable {randomGreeting} shouldn't be nested instead you should write it this way  randomGreeting as I don't get the random greetings from the backend API but when i change the variable the greetings show up
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @DJ-MrJay 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* No linter errors 👍 
* Redux setup properly 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Greeting.js (hunk 1 above)
Comment (DJ-MrJay) [reply src/Greeting.js]: Thanks for the tip!

**Review by roshan-bajgain — APPROVED** [review]
Hi @DJ-MrJay, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- Gitflow is followed. ✔️
- Good job on changing the requested✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**